### PR TITLE
use esbuild & JSR

### PR DESCRIPTION
**Pull request by BlowaterNostr**

### CODE REVIEW

File: .github/workflows/deno.yml [hunk 1]
```diff
@@ -57,13 +57,13 @@ jobs:
       - name: Clone repository
         uses: actions/checkout@v3
         with:
-          submodules: 'true'
+          submodules: true
```
Comment (bob2402) [inline .github/workflows/deno.yml:60]: Maybe can delete

File: deno.json [hunk 2]
```diff
@@ -33,12 +32,22 @@
             "*cov_profile*",
             "*tauri*",
             "app/UI/assets/",
-            "*/tailwind.js"
+            "*/tailwind.js",
+            "build-pwa/"
```
Comment (bob2402) [inline deno.json:36]: Why do we need to format the `build-pwa` folder even though it's listed in `gitignore`?

Comment (BlowaterNostr) [reply deno.json:36]: We don't need, that's why it's excluded. `.gitignore` and `deno fmt` are separate

Comment (bob2402) [reply deno.json:36]: Okay, I made a mistake.

**Review by bob2402 — APPROVED** [review]
LGTM

**Pull request merged**